src/nodes_io/writer_cinema_dng.rs

```diff
@@ -47,7 +47,11 @@ impl Parameterizable for CinemaDngWriter {
             .with("number-of-frames", Optional(IntRange(0, i64::MAX), ParameterValue::IntRange(0)))
     }
 
-    fn from_parameters(parameters: &Parameters, _context: &ProcessingContext) -> Result<Self>
+    fn from_parameters(
+        mut parameters: Parameters,
+        _is_input_to: &[NodeID],
+        _context: &ProcessingContext,
```

Comment:
maybe we want to actually split this trait so that sinks dont get passed an `_is_input_to`